capstone/src/main/webapp/js/autocomplete.js

```diff
@@ -115,8 +118,9 @@ function displaySuggestions(trimArr, autocompleteList, val) {
 
 /**
  * Event occurrance when input is provided to autocomplete field
+ * @param{Object} suggestions
```

Comment:
If you can find a more specific type than `Object` that would be nice. If not, then you can describe `suggestions` in a way that makes it's format clear. As it stands I'm not getting a clear idea of what `suggestions` is supposed to look like.